sections/Abstract.tex

```diff
@@ -0,0 +1,14 @@
+Being able to perform accurate time series forecasting has always played a big role for humanity, especially when it comes to predicting the weather. 
+Today, the state of the art methods for achieving the best possible results employ machine learning. 
+For many years, however, machine and deep learning techniques have had trouble dealing with large datasets. 
+More recently, the transformer model has shown great potential handling large datasets in the field of natural language processing.
```

Comment:
We make a few claims that may need cites?
1. Time series forecasting playing a big role (don't disagree, but still a claim)
2. Machine- and deep-learning models having trouble with large datasets (again, don't disagree, but still a claim)

Reply:
Added to supervisor questions